tests/api-controller-acceptance-tests/apicontroller/component_test.go

```diff
@@ -41,6 +41,124 @@ func TestComponentSpec(t *testing.T) {
 
 	Convey("API Controller should", t, func() {
 
+		Convey("update API with implicitly configured authentication to default authentication", func() {
+			t.Log("update API with implicitly configured authentication to default authentication")
+
+			testID := ctx.generateTestID(testIDLength)
+			t.Logf("Running test: %s", testID)
+
+			/*
+				  authentication:
+					- type: JWT
+					  jwt:
+						jwksUri: http://dex-service.kyma-system.svc.cluster.local:5556/keys
+						issuer: https://accounts.google.com
+			*/
+			api := ctx.apiFor(testID, domainName, namespace, apiSecurityDisabled, true)
+			ctx.setCustomJwtAuthenticationConfig(api)
+			api.Spec.AuthenticationEnabled = nil
+
+			createdAPI, err := kymaClient.GatewayV1alpha2().Apis(namespace).Create(api)
+			defer ctx.cleanUpAPI(kymaClient, api, t, true, namespace)
+			So(err, ShouldBeNil)
+			So(createdAPI, ShouldNotBeNil)
+			So(createdAPI.ResourceVersion, ShouldNotBeEmpty)
+
+			createdAPI, err = ctx.awaitAPIChanged(kymaClient, createdAPI, true, false, namespace)
+			So(err, ShouldBeNil)
```

Comment:
Please assert also on states of dependent Istio objects: Policy/VirtualService. In this way the test is more precisely defined - it's visible what is the expected state of the cluster in terms of our API and Istio API.